src/common/home/Home.jsx

```diff
@@ -94,9 +92,12 @@ const Home = () => {
                 </div>{" "}
               </span>
             </a>
-          </div>          
+          </div>
           <div className="video">
-            <YoutubeVideoEmbed videoId="XrWonrp296w" title="🤩 Introducing ReactPlay - An Opensource Platform to Create, Learn, and Share ReactJS Projects" />
+            <YoutubeVideoEmbed
```

Comment:
I think your prettify setting giving some extra spaces. You may want to check this one

https://discord.com/channels/982239924227031070/983221881723301899/1009049261826125844